doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -243,6 +243,14 @@ To enable modem sleep and TAU pre-warning notifications, use the following optio
 
 For additional configurations related to these features, see the API documentation.
 
+Limitations
+***********
+
+* The LTE link control library and the :ref:`nrfxlib:nrf_modem_at` interface should be used at the same time with caution
+  because the library also uses the :ref:`nrfxlib:nrf_modem_at` interface.
+  There are AT commands for which the library does not provide an API,
+  and the application needs to use the AT commands directly with the :ref:`nrfxlib:nrf_modem_at` interface.
```

Comment:
```suggestion
The LTE link control library and the :ref:`nrfxlib:nrf_modem_at` interface should be used at the same time with caution, because the library also uses the AT commands interface.
There are AT commands for which the library does not provide an API and the application needs to use the commands directly with the AT commands interface.
```
A bit too many links to the same target in the same paragraph. Also, this is only one item, so no bullets.

Reply:
I don't really see what the drawback is in using the links.
Advantage is that if the link target name is changed, all occurrences to it are changed also.
The only drawback is that it doesn't look as good on RST file. Generated document still looks good IMO.

Reply:
I'd still argue that too much is too much. The reader might wonder whether the links have some difference. Maybe you could even say just "the interface" instead of the full "AT commands interface".

Reply:
I was originally hoping that when you refactor the sentence you put "the interface" in places where it could be used. I've learned that using "it" and similar words may not refer to where I was intending to refer 😄
I need to modify the content of the sentences based on offline discussion with @tokangas so I'll add use of "the interface" too and you can then see how it looks.